addons/l10n_in_edi_stock/models/res_config_settings.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    l10n_in_edi_stock_username = fields.Char("Indian EDI Stock username",
+        related="company_id.l10n_in_edi_stock_username", readonly=False)
+    l10n_in_edi_stock_password = fields.Char("Indian EDI Stock password",
+        related="company_id.l10n_in_edi_stock_password", readonly=False)
+    l10n_in_edi_stock_production_env = fields.Boolean(
```

Comment:
We should reuse the production env from account.  It would be weird to have stock in test and invoice in production

Reply:
okay, got it. using from the account.